lib/floki/html/tree_construction.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Floki.HTML.TreeConstruction do
```

Comment:
Been a long road to get here- congrats!

Now the real "fun" starts. 😄

Reply:
@mischov thank you! 💜 😄  
I hope I can have some fun with all this complexity 😅